### ISSUE TITLE
Bad character escape sequence on 1.0.0-canary8 and Windows

### ISSUE THREAD

**Issue by andrehsu** [vinissimus/next-translate#385]
I encountered the following when running `next build`
```
info  - Creating an optimized production build
Failed to compile.
./pages/api/users/[userId]/emails/[emailId].ts:157:31
Syntax error: Bad character escape sequence
  155 |               ...ctx,
  156 | // @ts-ignore
> 157 |               pathname: '/api\users\[userId]\emails\[emailId]'
,
      |                                ^
  158 | // @ts-ignore
  159 |               loaderName: 'getServerSideProps',
  160 | // @ts-ignore
```

Using next-translate 1.0.0-canary8 on Windows.
This problem came up after migrating to 1.0.0, and reverting the changes fixed it.

**Comment by aralroca** [comment]
The weird thing is that it should not do anything if the route is `pages/api/*`, I guess it's for the wrong character... I'll take a look! Thanks to report it! 🙏

**Comment by aralroca** [comment]
And it is happening with 1.0.0-canary.7? I hope it is not something introduced in the last changes

**Comment by andrehsu** [comment]
@aralroca 
Oh sorry, it doesn't happen on canary.8, next-translate was declared using `^1.0.0`, and npm thinks 1.0.0-experimental.20 > 1.0.0-canary.8. Specifying 1.0.0-canary.8 fixes it.

**Comment by aralroca** [comment]
I'm glad to hear it. In fact, @vimutti77 fixed this in #362
